backend/Makefile

```diff
@@ -0,0 +1,52 @@
+# Commands
+runserver:
+	PIPENV_VERBOSITY=-1
+	pipenv run python manage.py runserver 0.0.0.0:8000
+
+# Run tests
+runtests:
+	PIPENV_VERBOSITY=-1
+	pipenv run python manage.py test
+
+
+# Launch docker database & cache
+launchdb:
+	docker compose up -d
+
+# Make migrations (when models are changed)
+makemigrations:
+	python manage.py makemigrations
+
+# Migrate (when migrations are created)
+migrate:
```

Comment:
I don't think we should abstract away the python manage.py xxx commands (see my comment on the readme)